src/userCLI/src/test/kotlin/org/imperial/mrc/hint/userCLI/AppTests.kt

```diff
@@ -1,53 +1,58 @@
 package org.imperial.mrc.hint.userCLI
 
-import org.imperial.mrc.hint.exceptions.UserException
 import org.assertj.core.api.Assertions
+import org.imperial.mrc.hint.db.DbConfig
+import org.imperial.mrc.hint.exceptions.UserException
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Test
-import org.junit.jupiter.api.extension.ExtendWith
-import org.springframework.boot.test.context.SpringBootTest
-import org.springframework.test.context.ActiveProfiles
-import org.springframework.test.context.junit.jupiter.SpringExtension
-import org.springframework.transaction.annotation.Transactional
-import org.springframework.beans.factory.annotation.Autowired
-import org.springframework.context.ApplicationContext
 
 //The hint db container must be running to run these tests
-@ActiveProfiles(profiles=["test"])
-@SpringBootTest
-@ExtendWith(SpringExtension::class)
-@Transactional
-class AppTests
-{
+class AppTests {
+
     companion object {
         const val TEST_EMAIL = "test@test.com"
+
+        val props = DatabaseProperties()
+        val dataSource = DbConfig().dataSource(props.url, props.user, props.password)
+        val sut = UserCLI(dataSource)
+
+        @AfterAll
+        @JvmStatic
+        fun cleanup() {
+            dataSource.connection.close()
+        }
     }
 
-    @Autowired
-    private lateinit var context: ApplicationContext
+    @BeforeEach
+    fun `remove user if exists`() {
```

Comment:
we no longer have spring magic "transactional" unfortunately, so just manually resetting the db each time